frontend/src/app/components/MainInstallButton.js

```diff
@@ -49,11 +56,8 @@ export default class MainInstallButton extends React.Component {
       </div>
     );
   }
-
-  install() {
-    const { eventCategory, hasAddon } = this.props;
-    if (hasAddon) { return; }
-    this.setState({ isInstalling: true });
-    installAddon(eventCategory);
-  }
 }
+
+MainInstallButton.contextTypes = {
```

Comment:
Okay, cool beans. Was worried I missed something, but nope. Let's save this for future work